src/main/resources/application.properties

```diff
@@ -26,7 +26,7 @@ db.stairway.forceClean=true
 oauth.clientId=970791974390-1581mjhtp2b3jmg4avhor1vabs13b7ur.apps.googleusercontent.com
 oauth.schemeName=googleoauth
 oauth.loginEndpoint=https://accounts.google.com/o/oauth2/auth
-oauth.scopes=openid,email,profile
+oauth.scopes=openid,email,profile,https://www.googleapis.com/auth/cloud-platform
```

Comment:
I wonder why this is in `application.properties`. Probably just historical.
These values are required by the code. It is not like we change this across different deployments.
This config is only used to feed SwaggerDocumentationConfig which looks like generated code that is no longer generated. Hmm...

Reply:
As I recall, swagger-ui actually needs this to have this scope appear.